src/main/resources/assets/admin/common/js/data/PropertySet.ts

```diff
@@ -461,6 +461,38 @@ module api.data {
             return true;
         }
 
+        public syncEmptyArrays(target: PropertySet) {
+
+            if(!target) {
+                return;
+            }
+
+            target.getPropertyArrays().forEach((propertyArray: PropertyArray) => {
+                if (!this.getPropertyArray(propertyArray.getName())) {
+
+                    this.addPropertyArray(propertyArray.copy(this));
+                }
+
+                propertyArray.forEach(((property, index) => {
+                    if (property.getType().equals(api.data.ValueTypes.DATA)) {
+
+                        const targetPropertySet = property.getValue().getPropertySet();
+
```

Comment:
``PropertyValue`` class has ``getPropertySet()`` method, so you can do ``property.getPropertySet()`` directly.